Source/com/Communicator.h

```diff
@@ -776,6 +784,11 @@ namespace RPC {
             }
 
         public:
+            void ForcedDestructionTimes(const uint8_t softKillCheckWaitTime, const uint8_t hardKillCheckWaitTime)
```

Comment:
Perhaps I'm missing something but is this even called?

Reply:
It is used from PluginServer.h

Reply:
Sorry it was redundant and removed